src/commands/synthetics/batch.ts

```diff
@@ -268,6 +274,31 @@ const getResultFromBatch = (
   }
 }
 
+const getResultWithoutPollResult = (
+  resultInBatch: BaseResultInBatch,
+  test: Test,
+  hasTimedOut: boolean,
+  resultDisplayInfo: ResultDisplayInfo
+): Result => {
+  const {getLocation, options} = resultDisplayInfo
+
+  return {
+    executionRule: resultInBatch.execution_rule,
+    initialResultId: resultInBatch.initial_result_id,
+    isNonFinal: isNonFinalResult(resultInBatch),
+    location: getLocation(resultInBatch.location, test),
+    passed: hasResultPassed(resultInBatch, false, hasTimedOut, options),
+    result: {} as ServerResult,
```

Comment:
If we don't need this, could we rather make it optional?

Reply:
I'm afraid of how it could break customer's code. I'll check internally (web-ui and CI integrations)